tests/conftest.py

```
@@ -1,7 +1,55 @@
+"""Test configuration and fixtures."""
 import pytest
-from fastapi.testclient import TestClient
-from backend.main import app
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)